dist/stat_data.py

```diff
@@ -359,9 +359,9 @@ def __init__(self, name, desc, flags=''):
     EvictStat('eviction_slow', 'eviction server unable to reach eviction goal'),
     EvictStat('eviction_stable_state_workers', 'eviction worker thread stable number', 'no_clear'),
     EvictStat('eviction_state', 'eviction state', 'no_clear,no_scale'),
-    EvictStat('eviction_target_strategy_both_clean_and_dirty', 'eviction walk target strategy both clean and dirty pages'),
     EvictStat('eviction_target_strategy_clean', 'eviction walk target strategy only clean pages'),
     EvictStat('eviction_target_strategy_dirty', 'eviction walk target strategy only dirty pages'),
+    EvictStat('eviction_target_strategy_updates', 'eviction walk target strategy updated pages'),
```

Comment:
I feel we do not need `only` anymore. I feel `eviction walk target strategy clean pages` and `eviction walk target strategy dirty pages` would be better. 

The third one is actually `pages with updates` would be better. `eviction walk target strategy pages with updates` because the `updated pages` is some how similar to `dirty pages`.

Opinions?

Reply:
updated